### PR TITLE
Policy domains filter cover the TPs APIs

### PR DESCRIPTION
**Pull request by dbMundada**

### CODE REVIEW

File: src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala
Comment (khemrajrathore) [inline src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala:235]: @dbMundada This feels little hacky approach to me, Can we not take `regex` as input for domain, and match it here?

Also if you still want to move ahead with above proposed solution, make this a function, add exception handling and write some unit test around this

Comment (pandurangpatil) [reply src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala:235]: @dbMundada 
I second the inputs given by the @khemrajrathore to take it as a `regex` input through config folder rules. Also, add respective unit tests for the same.

**Review by pandurangpatil — CHANGES_REQUESTED** [review]

File: src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala
Comment (dbMundada) [reply src/main/scala/ai/privado/policyEngine/PolicyExecutor.scala:235]: sure, thanks for input. regex will cover one use case which was partially missing. Updating pr...

**Review by khemrajrathore — DISMISSED** [review]
LGTM, check are still red, once Green can be merged

**Pull request merged**
